### PR TITLE
Add documents for `information_schema.tidb_index_usage` and `sys.schema_unused_index`.

### PR DESCRIPTION
**Pull request by YangKeao**
### What is changed, added or deleted? (Required)

Add the document for index usage related system tables and databases.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16446
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-tidb-index-usage.md
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:1]: Please add this file to TOC.md

File: TOC.md [hunk 1]
```diff
@@ -977,6 +977,7 @@
       - PERFORMANCE_SCHEMA
         - [概述](/performance-schema/performance-schema.md)
         - [`SESSION_CONNECT_ATTRS`](/performance-schema/performance-schema-session-connect-attrs.md)
+      - [`sys`](/sys-schema.md)
```
Comment (Oreoxmt) [inline TOC.md:980]:
```suggestion
      - [`SYS`](/sys-schema.md)
```

File: information-schema/information-schema-tidb-index-usage.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_INDEX_USAGE
+summary: 了解 information_schema 表 `TIDB_INDEX_USAGE`。
+---
+
+# TIDB_INDEX_USAGE
+
+`TIDB_INDEX_USAGE` 记录了当前节点所有 Index 的访问统计信息。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:8]:
```suggestion
TiDB 从 v8.0.0 开始提供 `TIDB_INDEX_USAGE` 表，你可以使用该表查看当前 TiDB 节点中所有索引的访问统计信息。
```

File: information-schema/information-schema-tidb-index-usage.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+{{< copyable "sql" >}}
+
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:11]:
```suggestion
```
This copy syntax is no longer needed for the current docs website. Please remove it from this doc.

File: information-schema/information-schema-tidb-index-usage.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+```sql
+USE information_schema;
+DESC tidb_index_usage;
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:14]:
```suggestion
USE INFORMATION_SCHEMA;
DESC TIDB_INDEX_USAGE;
```

File: information-schema/information-schema-tidb-index-usage.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* `KV_REQ_TOTAL`：访问该索引时产生的 KV 请求总数。
+* `ROWS_ACCESS_TOTAL`：访问该索引时扫描的总行数。
+* `PERCENTAGE_ACCESS_0`：访问行数占表总行数比例为 0 的次数。
+* `PERCENTAGE_ACCESS_0_1`：访问行数占总行数比例为 0 到 1% 的次数。
+* `PERCENTAGE_ACCESS_1_10`：访问行数占总行数比例为 1% 到 10% 的次数。
+* `PERCENTAGE_ACCESS_10_20`：访问行数占总行数比例为 10% 到 20% 的次数。
+* `PERCENTAGE_ACCESS_20_50`：访问行数占总行数比例为 20% 到 50% 的次数。
+* `PERCENTAGE_ACCESS_50_100`：访问行数占总行数比例为 50% 到 100% 的次数。
+* `PERCENTAGE_ACCESS_100`：访问行数占总行数比例为 100% 的次数。
+* `LAST_ACCESS_TIME`：最后一次访问该索引的时间。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:54]:
```suggestion
* `INDEX_NAME`：索引的名称。
* `QUERY_TOTAL`：访问该索引的语句总数。
* `KV_REQ_TOTAL`：访问该索引时产生的 KV 请求总数。
* `ROWS_ACCESS_TOTAL`：访问该索引时扫描的总行数。
* `PERCENTAGE_ACCESS_0`：行访问比例（访问行数占表总行数的百分比）为 0 的次数。
* `PERCENTAGE_ACCESS_0_1`：行访问比例为 0 到 1% 的次数。
* `PERCENTAGE_ACCESS_1_10`：行访问比例为 1% 到 10% 的次数。
* `PERCENTAGE_ACCESS_10_20`：行访问比例为 10% 到 20% 的次数。
* `PERCENTAGE_ACCESS_20_50`：行访问比例为 20% 到 50% 的次数。
* `PERCENTAGE_ACCESS_50_100`：行访问比例为 50% 到 100% 的次数。
* `PERCENTAGE_ACCESS_100`：行访问比例为 100% 的次数。
* `LAST_ACCESS_TIME`：最近一次访问该索引的时间。
```

File: information-schema/information-schema-tidb-index-usage.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:55]:
````suggestion

## CLUSTER_TIDB_INDEX_USAGE

`TIDB_INDEX_USAGE` 表仅提供单个 TiDB 节点中所有索引的访问统计信息。如果要查看整个集群上所有 TiDB 节点中索引的访问统计信息，需要查询 `TIDB_INDEX_USAGE` 表。

```sql
USE INFORMATION_SCHEMA;
DESC TIDB_INDEX_USAGE;
```

````

Refer to https://docs.pingcap.com/zh/tidb/stable/information-schema-tidb-trx#cluster_tidb_trx

File: information-schema/information-schema-tidb-index-usage.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+# 限制
+
+- `TIDB_INDEX_USAGE` 表中的数据可能有至多 5 分钟的延迟。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:58]:
```suggestion
## 使用限制

- `TIDB_INDEX_USAGE` 表中的数据可能存在最多 5 分钟的延迟。
```

File: sys-schema.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+---
+title: sys Schema
+summary: 了解 sys 库中的系统表。
```
Comment (Oreoxmt) [inline sys-schema.md:3]:
```suggestion
summary: 了解 TiDB `sys` 系统数据库。
```

File: sys-schema.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+>     table_name as object_name,
+>     index_name
+>   FROM information_schema.cluster_tidb_index_usage
+>   WHERE
+>     table_schema not in ('sys', 'mysql', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA') and
+>     index_name != 'PRIMARY'
+>   GROUP BY table_schema, table_name, index_name
+>   HAVING
+>     sum(last_access_time) is null;
+> ```
```
Comment (Oreoxmt) [inline sys-schema.md:28]:
````suggestion
TiDB 从 v8.0.0 开始提供 `sys` Schema。你可以通过查看 `sys` 系统数据库中的表或视图理解 TiDB 的系统表、[`INFORMATION_SCHEMA` 表](/information-schema/information-schema.md)和 [`PERFORMANCE SCHEMA` 表](/performance-schema/performance-schema.md)内的数据。

对于从 v8.0.0 之前版本升级的集群，`sys` Schema 和其中的视图不会自动创建。你可以通过以下 SQL 语句手动创建：

```sql
CREATE DATABASE IF NOT EXISTS sys;
CREATE OR REPLACE VIEW sys.schema_unused_indexes AS
  SELECT
    table_schema as object_schema,
    table_name as object_name,
    index_name
  FROM information_schema.cluster_tidb_index_usage
  WHERE
    table_schema not in ('sys', 'mysql', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA') and
    index_name != 'PRIMARY'
  GROUP BY table_schema, table_name, index_name
  HAVING
    sum(last_access_time) is null;
```
````

File: sys-schema.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+`sys` 库里的表或视图被用于帮助用户理解 TiDB 的系统表、`INFORMATION_SCHEMA` 表和 `Performance Schema` 表内的数据。
+
```
Comment (Oreoxmt) [inline sys-schema.md:31]:
```suggestion
```

File: sys-schema.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+- `schema_unused_index` 用于记录自从 TiDB 上一次重启以来未使用的索引。
+    - `OBJECT_SCHEMA`：索引所在表的所属数据库的名称。
+    - `OBJECT_NAME`：索引所在表的名称。
+    - `INDEX_NAME`：索引的名称。
```
Comment (Oreoxmt) [inline sys-schema.md:35]:
```suggestion
## `schema_unused_index`

`schema_unused_index` 用于记录自 TiDB 上次启动以来未被使用的索引信息，包括如下列：

- `OBJECT_SCHEMA`：索引所在表的所属数据库的名称。
- `OBJECT_NAME`：索引所在表的名称。
- `INDEX_NAME`：索引的名称。
```

Comment (Oreoxmt) [reply sys-schema.md:35]: To further enhance the comprehensiveness, we can include the following commands or add usage examples:

```
USE sys;
DESC schema_unused_indexes;
+---------------+-------------+------+------+---------+-------+
| Field         | Type        | Null | Key  | Default | Extra |
+---------------+-------------+------+------+---------+-------+
| object_schema | varchar(64) | YES  |      | NULL    |       |
| object_name   | varchar(64) | YES  |      | NULL    |       |
| index_name    | varchar(64) | YES  |      | NULL    |       |
+---------------+-------------+------+------+---------+-------+
3 rows in set (0.01 sec)
```

File: information-schema/information-schema-tidb-index-usage.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+`TIDB_INDEX_USAGE` 表中列的含义如下：
+
+* `TABLE_SCHEMA`：索引所在表的所属数据库的名称。
+* `TABLE_NAME`：索引所在表的名称。
+* `INDEX_NAME`：索引的名称。
+* `QUERY_TOTAL`：访问该索引的语句总数。
+* `KV_REQ_TOTAL`：访问该索引时产生的 KV 请求总数。
+* `ROWS_ACCESS_TOTAL`：访问该索引时扫描的总行数。
```
Comment (qw4990) [inline information-schema/information-schema-tidb-index-usage.md:44]: 对于大表，这个字段一直累加会不会有溢出的风险？需要特殊说明吗

**Review by qw4990 — APPROVED** [review]
REST LGTM

File: information-schema/information-schema-tidb-index-usage.md (hunk 12 above)
Comment (YangKeao) [reply information-schema/information-schema-tidb-index-usage.md:44]: 假设集群每秒钟总共扫描 0xFFFFFFFF 行，uint64 在 0xFFFFFFFF s = 136.19252 年的时候才会溢出，现在看应该没啥风险 :thinking: 。

**Review by Oreoxmt — APPROVED** [review]

File: information-schema/information-schema-tidb-index-usage.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
+---
+title: TIDB_INDEX_USAGE
+summary: 了解 information_schema 表 `TIDB_INDEX_USAGE`。
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-index-usage.md:3]:
```suggestion
summary: 了解 INFORMATION_SCHEMA 表 `TIDB_INDEX_USAGE`。
```

File: information-schema/information-schema-tidb-index-usage.md [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+* `PERCENTAGE_ACCESS_20_50`：行访问比例为 20% 到 50% 的次数。
+* `PERCENTAGE_ACCESS_50_100`：行访问比例为 50% 到 100% 的次数。
+* `PERCENTAGE_ACCESS_100`：行访问比例为 100% 的次数。
+* `LAST_ACCESS_TIME`：最近一次访问该索引的时间。
+
+## CLUSTER_TIDB_INDEX_USAGE
+
+`TIDB_INDEX_USAGE` 表仅提供单个 TiDB 节点中所有索引的访问统计信息。如果要查看整个集群上所有 TiDB 节点中索引的访问统计信息，需要查询 `CLUSTER_TIDB_INDEX_USAGE` 表。
+
+与 `TIDB_INDEX_USAGE` 表的查询结果相比，`CLUSTER_TIDB_INDEX_USAGE` 表的查询结果额外包含了 INSTANCE 字段。INSTANCE 字段展示了集群中各节点的 IP 地址和端口，用于区分不同节点上的统计信息。
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-index-usage.md:58]:
```suggestion
与 `TIDB_INDEX_USAGE` 表的查询结果相比，`CLUSTER_TIDB_INDEX_USAGE` 表的查询结果额外包含了 `INSTANCE` 字段。`INSTANCE` 字段展示了集群中各节点的 IP 地址和端口，用于区分不同节点上的统计信息。
```

File: information-schema/information-schema.md [hunk 15]
```diff
@@ -94,6 +95,7 @@ Information Schema 提供了一种查看系统元数据的 ANSI 标准方法。
 | [`TIDB_INDEXES`](/information-schema/information-schema-tidb-indexes.md)                | 提供有关 TiDB 表的索引信息。                                   |
 | [`TIDB_SERVERS_INFO`](/information-schema/information-schema-tidb-servers-info.md)      | 提供 TiDB 服务器的列表                                         |
 | [`TIDB_TRX`](/information-schema/information-schema-tidb-trx.md)                        | 提供 TiDB 节点上正在执行的事务的信息。                         |
+| [`TIDB_INDEX_USAGE`](/information-schema/information-schema-tidb-index-usage.md)        | 提供 TiDB 节点上有关访问索引的统计信息。 ｜
```
Comment (hfxsd) [inline information-schema/information-schema.md:98]: 按字母顺序排
```suggestion
| [`TIDB_INDEXES`](/information-schema/information-schema-tidb-indexes.md)                | 提供有关 TiDB 表的索引信息。                                   |
| [`TIDB_INDEX_USAGE`](/information-schema/information-schema-tidb-index-usage.md)        | 提供 TiDB 节点上有关访问索引的统计信息。 ｜
| [`TIDB_SERVERS_INFO`](/information-schema/information-schema-tidb-servers-info.md)      | 提供 TiDB 服务器的列表                                         |
| [`TIDB_TRX`](/information-schema/information-schema-tidb-trx.md)                        | 提供 TiDB 节点上正在执行的事务的信息。                         |
```

File: sys-schema.md [hunk 16]
```diff
@@ -0,0 +1,52 @@
+---
+title: sys Schema
+summary: 了解 TiDB `sys` 系统数据库。
+---
+
+# `sys` Schema
+
+TiDB 从 v8.0.0 开始提供 `sys` Schema。你可以通过查看 `sys` 系统数据库中的表或视图理解 TiDB 的系统表、[`INFORMATION_SCHEMA` 表](/information-schema/information-schema.md)和 [`PERFORMANCE SCHEMA` 表](/performance-schema/performance-schema.md)内的数据。
+
```
Comment (hfxsd) [inline sys-schema.md:9]:
```suggestion

## 手动创建 `sys` Schema 和视图

```

File: information-schema/information-schema.md [hunk 17]
```diff
@@ -71,6 +71,7 @@ Information Schema 提供了一种查看系统元数据的 ANSI 标准方法。
 | `CLUSTER_STATEMENTS_SUMMARY`                                                            | 提供 `STATEMENTS_SUMMARY` 表的集群级别的视图。                 |
 | `CLUSTER_STATEMENTS_SUMMARY_HISTORY`                                                    | 提供 `STATEMENTS_SUMMARY_HISTORY` 表的集群级别的视图。         |
 | `CLUSTER_TIDB_TRX`                                                                      | 提供 `TIDB_TRX` 表的集群级别的视图。                           |
+| `CLUSTER_TIDB_INDEX_USAGE`                                                                      | 提供 `TIDB_INDEX_USAGE` 表的集群级别的视图。                           |
```
Comment (hfxsd) [inline information-schema/information-schema.md:74]: 按字母顺序排
```suggestion
| `CLUSTER_TIDB_INDEX_USAGE`                                                              | 提供 `TIDB_INDEX_USAGE` 表的集群级别的视图。                           |
| `CLUSTER_TIDB_TRX`                                                                      | 提供 `TIDB_TRX` 表的集群级别的视图。                           |
```

File: sys-schema.md [hunk 18]
```diff
@@ -0,0 +1,52 @@
+---
+title: sys Schema
+summary: 了解 TiDB `sys` 系统数据库。
+---
+
+# `sys` Schema
+
+TiDB 从 v8.0.0 开始提供 `sys` Schema。你可以通过查看 `sys` 系统数据库中的表或视图理解 TiDB 的系统表、[`INFORMATION_SCHEMA` 表](/information-schema/information-schema.md)和 [`PERFORMANCE SCHEMA` 表](/performance-schema/performance-schema.md)内的数据。
```
Comment (hfxsd) [inline sys-schema.md:8]:
```suggestion
TiDB 从 v8.0.0 开始提供 `sys` Schema。你可以通过查看 `sys` 系统数据库中的表或视图理解 TiDB 的系统表、[`INFORMATION_SCHEMA`](/information-schema/information-schema.md) 表和 [`PERFORMANCE SCHEMA`](/performance-schema/performance-schema.md) 表内的数据。
```

**Review by hfxsd — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

File: information-schema/information-schema-tidb-index-usage.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
+---
+title: TIDB_INDEX_USAGE
+summary: 了解 INFORMATION_SCHEMA 表 `TIDB_INDEX_USAGE`。
+---
+
+# TIDB_INDEX_USAGE
+
+TiDB 从 v8.0.0 开始提供 `TIDB_INDEX_USAGE` 表，你可以使用该表查看当前 TiDB 节点中所有索引的访问统计信息。在 SQL 语句执行时，将默认维护访问索引有关的统计信息，可以通过修改配置项 [`instance.tidb_enable_collect_execution_info`](/tidb-configuration-file.md#tidb_enable_collect_execution_info) 或者变量[`tidb_enable_collect_execution_info`](/system-variables.md#tidb_enable_collect_execution_info) 将其关闭。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-index-usage.md:8]:
```suggestion
TiDB 从 v8.0.0 开始提供 `TIDB_INDEX_USAGE` 表，你可以使用该表查看当前 TiDB 节点中所有索引的访问统计信息。在 SQL 语句执行时，TiDB 默认维护访问索引有关的统计信息，可以通过修改配置项 [`instance.tidb_enable_collect_execution_info`](/tidb-configuration-file.md#tidb_enable_collect_execution_info) 或者系统变量[`tidb_enable_collect_execution_info`](/system-variables.md#tidb_enable_collect_execution_info) 将其关闭。
```
请在 `instance.tidb_enable_collect_execution_info` 和 `tidb_enable_collect_execution_info` 的描述中添加 TIDB_INDEX_USAGE 的说明

File: information-schema/information-schema.md [hunk 20]
```diff
@@ -70,6 +70,7 @@ Information Schema 提供了一种查看系统元数据的 ANSI 标准方法。
 | `CLUSTER_SLOW_QUERY`                                                                    | 提供 `SLOW_QUERY` 表的集群级别的视图。                         |
 | `CLUSTER_STATEMENTS_SUMMARY`                                                            | 提供 `STATEMENTS_SUMMARY` 表的集群级别的视图。                 |
 | `CLUSTER_STATEMENTS_SUMMARY_HISTORY`                                                    | 提供 `STATEMENTS_SUMMARY_HISTORY` 表的集群级别的视图。         |
+| `CLUSTER_TIDB_INDEX_USAGE`                                                              | 提供 `TIDB_INDEX_USAGE` 表的集群级别的视图。                           |
 | `CLUSTER_TIDB_TRX`                                                                      | 提供 `TIDB_TRX` 表的集群级别的视图。                           |
 | [`CLUSTER_SYSTEMINFO`](/information-schema/information-schema-cluster-systeminfo.md)    | 提供集群中服务器的内核参数配置的详细信息。                     |
```
Comment (Oreoxmt) [inline information-schema/information-schema.md:75]:
```suggestion
| [`CLUSTER_SYSTEMINFO`](/information-schema/information-schema-cluster-systeminfo.md)    | 提供集群中服务器的内核参数配置的详细信息。                     |
| `CLUSTER_TIDB_INDEX_USAGE`                                                              | 提供 `TIDB_INDEX_USAGE` 表的集群级别的视图。                           |
| `CLUSTER_TIDB_TRX`                                                                      | 提供 `TIDB_TRX` 表的集群级别的视图。                           |
```

**Review by Oreoxmt — APPROVED** [review]

File: information-schema/information-schema-tidb-index-usage.md [hunk 21]
```diff
@@ -0,0 +1,93 @@
+---
+title: TIDB_INDEX_USAGE
+summary: 了解 INFORMATION_SCHEMA 表 `TIDB_INDEX_USAGE`。
+---
+
+# TIDB_INDEX_USAGE
+
+TiDB 从 v8.0.0 开始提供 `TIDB_INDEX_USAGE` 表，你可以使用该表查看当前 TiDB 节点中所有索引的访问统计信息。在 SQL 语句执行时，TiDB 默认维护访问索引有关的统计信息，可以通过修改配置项 [`instance.tidb_enable_collect_execution_info`](/tidb-configuration-file.md#tidb_enable_collect_execution_info) 或者系统变量[`tidb_enable_collect_execution_info`](/system-variables.md#tidb_enable_collect_execution_info) 将其关闭。
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-index-usage.md:8]:
```suggestion
TiDB 从 v8.0.0 开始提供 `TIDB_INDEX_USAGE` 表，你可以使用该表查看当前 TiDB 节点中所有索引的访问统计信息。在 SQL 语句执行时，TiDB 默认维护访问索引有关的统计信息，可以通过修改配置项 [`instance.tidb_enable_collect_execution_info`](/tidb-configuration-file.md#tidb_enable_collect_execution_info) 或者系统变量 [`tidb_enable_collect_execution_info`](/system-variables.md#tidb_enable_collect_execution_info) 将其关闭。
```

**Pull request merged**